.github/workflows/docs.yml

```diff
@@ -0,0 +1,26 @@
+name: Documentation
+
+on:
+  push:
+    branches: [ master ]
+
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  bundle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v1
+
+      - name: build documentation
+        run: cargo doc --lib --no-deps --all-features --open
```

Comment:
why does it have `--no-deps`?
Also, `--open` seems wrong here, we don't want to open them in a browser on CI :)

Reply:
d'oh!

Reply:
fixed